modules/core/src/lib/attribute/data-column.ts

```diff
@@ -509,20 +520,20 @@ export default class DataColumn<Options, State> {
   // https://developer.mozilla.org/en-US/docs/Web/API/WebGLRenderingContext/vertexAttribPointer
   normalizeConstant(value: NumericArray): NumericArray {
```

Comment:
Should this always return a `Float32Array`?

If we are using luma.gl types, should this be a luma.gl utility?